runelite-api/src/main/java/net/runelite/api/Client.java

```diff
@@ -1715,6 +1715,16 @@
 
 	void draw2010Menu();
 
+	/**
+	 * A menu that looks like the original but has support for alpha values
```

Comment:
I'd prefer this be improved or excluded. Perhaps something along these lines? (please check that this is valid javadoc before committing, I didn't :sweat_smile: )
```java
/**
 * Draws a menu in the OSRS style at the mouse cursor's position with any client menu settings
 * applied.
 *
 * {@see setMenuAlpha(int alpha)}
 */
```

In conjunction with my other comment below, this is a more concise